openwebstart/src/main/java/com/openwebstart/jvm/ui/dialogs/ConfigurationDialog.java

```diff
@@ -316,11 +327,11 @@ private String getDefaultServer() {
 
     private class WarningToolTip extends JToolTip {
 
-            public WarningToolTip(JComponent component) {
-                super();
-                setComponent(component);
-                setBackground(Color.white);
-                setForeground(Color.decode("#f57c00"));
-            }
+        public WarningToolTip(JComponent component) {
+            super();
+            setComponent(component);
+            setBackground(Color.white);
+            setForeground(Color.decode("#f57c00"));
```

Comment:
I ask myself why we define custom foreground / background color. Since we use system look and feel this can end in a unpersitent look.